### PR TITLE
Whats New For phpBB 3.3 For Ext Authors

### PR DESCRIPTION
**Pull request by VSEphpbb**
If I missed anything, let me know.

If I was wrong about anything, let me know.

### CODE REVIEW

File: development/extensions/new_in_rhea_3.3.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        if (!$enableable && phpbb_version_compare(PHPBB_VERSION, '3.3.0', '>='))
+        {
+            return array('phpBB 3.2.0 or newer is required to enable this extension.');
+        }
+
+        // Return the result of the test, which could be true (or false for phpBB 3.2 and 3.1)
+        return $enableable;
+    }
+
+Also note that in this one case, we are hard-coding language in the extension. Technically, if your extension could not be enabled, then its language files will not be loaded in the system so using language keys here will not work. There are workarounds to this, however in the interest of cleaner and more elegant code an exception can be made here for allowing hard-coded language strings.
```
Comment (Crizz0) [inline development/extensions/new_in_rhea_3.3.rst]: Should we mention that they should use English in that hard-coding case?

Comment (paul999) [reply development/extensions/new_in_rhea_3.3.rst]: tbh, I disagree with the exception, we always have required stuff to be translated, except when the language absolutly wasn't being able to be loaded. Cleaner code is not a reason to hardcode any language IMO

Comment (VSEphpbb) [reply development/extensions/new_in_rhea_3.3.rst]: Then I will remove the text about cleaner code.

File: development/extensions/new_in_rhea_3.3.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+If you intend to start using some of the new language constructs introduced in PHP 7, you must make your extension's minimum PHP requirement known to your users. This can include setting the minimum PHP version in your extension's ``composer.json`` file as well as designating phpBB 3.3 as a minimum requirement. You can also use the ``ext.php`` file to check that the minimum PHP and phpBB version requirements are satisfied in the ``is_enableable()`` method, which will prevent users who do not meet the requirements from accidentally installing your extension. Examples of this can be found `here <tutorial_advanced.html#using-installation-commands-in-ext-php>`_.
+
+Symfony 3.4
+===========
+
+Symfony has been updated to 3.4 (from 2.8). The following changes are due to deprecations introduced in Symfony 2 that have been removed from Symfony 3. The following changes are **required** for phpBB 3.3 and later.
+
+.. note::
+    Although the following changes are required for phpBB 3.3, they are backwards compatible and therefor, will still work with phpBB 3.2 and 3.1.
```
Comment (marc1706) [inline development/extensions/new_in_rhea_3.3.rst]: I think you meant `therefore`

**Comment by VSEphpbb** [comment]
Updated for @marc1706 and @paul999 of the Jersey shore.

**Pull request merged**